android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMPBrowserActivity.java

```diff
@@ -0,0 +1,143 @@
+/**
+ * Copyright (C) 2020 SIL International. All rights reserved.
+ */
+
+package com.tavultesoft.kmea;
+
+import android.annotation.SuppressLint;
+import android.content.Context;
+import android.content.Intent;
+import android.graphics.Bitmap;
+import android.graphics.Color;
+import android.graphics.PorterDuff.Mode;
+import android.graphics.drawable.Drawable;
+import android.net.Uri;
+import android.os.Build;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.ImageButton;
+import android.webkit.WebChromeClient;
+import android.webkit.WebSettings;
+import android.webkit.WebView;
+import android.webkit.WebViewClient;
+import androidx.appcompat.app.AppCompatActivity;
+
+public class KMPBrowserActivity extends AppCompatActivity {
+
+  private WebView webView;
+  private static final String KMP_SEARCH_BASE_URL = "https://keyman.com/keyboards";
```

Comment:
Can you combine these into one format string, e.g. `KMP_SEARCH_URL_FORMATSTR="https://keyman.com/keyboards%s?embed=linux&version=%s"` and then use that format string around line 93?